ray-operator/test/e2e/create_detached_actor.py

```diff
@@ -0,0 +1,9 @@
+import ray
+import sys
+
+@ray.remote(num_cpus=1)
+class Actor:
+  pass
```

Comment:
Maybe the indentation width should be consistent with other python files? That is, 4 spaces.

Reply:
Good catch. Thanks. https://github.com/ray-project/kuberay/pull/2168/commits/f883b164a2e8ad3c3a7b10921b8209948d69eabc